controller/src/beerocks/bml/bml.cpp

```diff
@@ -709,3 +709,44 @@ int bml_start_dcs_single_scan(BML_CTX ctx, const char *radio_mac, int dwell_time
     return pBML->start_dcs_single_scan(tlvf::mac_from_string(std::string(radio_mac)), dwell_time,
                                        channel_pool, channel_pool_size);
 }
+
+int bml_client_get_client_list(BML_CTX ctx, char *client_list, unsigned int *client_list_size)
+{
+    // Validate input parameters
+    if (!ctx && !client_list && !client_list_size) {
```

Comment:
your condition is backward here:
```suggestion
    if (!ctx || !client_list || !client_list_size) {
```

you can also check separately:
```
if (*client_list_size == 0) {
    //handle error
}
```

Reply:
Updated the condition & added the size validation